src/main/java/core/basesyntax/Application.java

```diff
@@ -3,5 +3,12 @@
 public class Application {
     public static void main(String[] args) {
         // create three balls using class Lottery and print information about them in console
+        Ball ball1 = Lottery.getRandomBall();
+        Ball ball2 = Lottery.getRandomBall();
+        Ball ball3 = Lottery.getRandomBall();
```

Comment:
![image](https://github.com/mate-academy/jv-lottery/assets/20326858/de7e589d-fd97-451e-b43d-22d3be130824)

Reply:
Done, but I accidentally fumbled my pull request while trying to commit the changes (still learning my git terminal).
New pull is at https://github.com/mate-academy/jv-lottery/pull/1306
I'm very sorry for the inconvenience.